api/rpc/stat/stat.proto

```diff
@@ -3,13 +3,14 @@ syntax = "proto3";
 package stat;
 
 service Stat {
-  rpc GetMeasurments (MeasurementRequest) returns (MeasurementReply) {}
+  rpc ExecuteQuery (ExecuteQueryRequest) returns (ExecuteQueryReply) {}
 }
 
-message MeasurementRequest {
+message ExecuteQueryRequest {
```

Comment:
Just nitpicking here, but ExecuteQueryRequest strikes me as a tad verbose. Could the messages be shortened to QueryRequest and QueryReply?